apps/crn-frontend/src/shared-research/export.ts

```diff
@@ -75,7 +75,9 @@ export const researchOutputToCSV = (
   rrid: output.rrid,
   accession: output.accession,
   labCatalogNumber: output.labCatalogNumber,
-  description: htmlToCsvText(output.description),
+  description: [htmlToCsvText(output.description), output.descriptionMD]
```

Comment:
Some concern here. I think this function stripped out HTML from description. If this is now MD is this still required? I would have thought MD is just readable in exports. Maybe a question to tiff.

Reply:
I'm still cleaning up the old description if it exists, and adding the raw markdown next to it.